tests/examples/llm/common/conftest.py

```diff
@@ -13,19 +13,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+import sys
+
 import pytest
 
+from _utils import TEST_DIRS
 from _utils import import_or_skip
 
 
+@pytest.fixture(scope="module")
+def import_utils():
+    utils_path = os.path.join(TEST_DIRS.examples_dir, 'llm/common/')
+    sys.path.insert(0, utils_path)
+
+    import utils
+
+    return utils
+
+
+@pytest.fixture(scope="module")
+def import_web_scraper_module():
+    web_scraper_path = os.path.join(TEST_DIRS.examples_dir, 'llm/common/')
+    sys.path.insert(0, web_scraper_path)
+
+    import web_scraper_module
+
+    return web_scraper_module
+
+
+# Fixture for importing the module
+@pytest.fixture(scope="module")
+def import_content_extractor_module():
+    sys.path.insert(0, os.path.join(TEST_DIRS.examples_dir, 'llm/common'))
+
+    import content_extractor_module
+
+    return content_extractor_module
```

Comment:
Not sure if these are needed, as the `import_mod` already does this, usage would look like:
```
@pytest.mark.import_mod(os.path.join(TEST_DIRS.examples_dir, 'llm/common/web_scraper_module.py'))
```

Importantly if other examples contain a `utils.py` it would create a problem as this leaves `utils` in `sys.modules`, while the `import_mod` fixture explicitly pops the name after the test.